com.valvesoftware.Steam.yml

```diff
@@ -180,6 +180,7 @@ modules:
       - |
         set -e
         sed -i s:/usr/bin/steam:/app/bin/steam: steam.desktop
+        desktop-file-edit --remove-key=PrefersNonDefaultGPU steam.desktop
         desktop-file-edit --remove-key=X-KDE-RunOnDiscreteGpu steam.desktop
```

Comment:
Am I understanding correctly that these two lines from fdbd21c are a workaround for https://github.com/ValveSoftware/steam-for-linux/issues/9383?

Reply:
Yes, I mentioned the PR it on that issue, it's pretty close to the top.

Reply:
Ah, I see it now. Thank you for the clarification.

Here's the full context so that future readers don't need to chase it down:

PR #1172 re-applies changes from fdbd21c. These changes were accidentally reverted in a25bdf1f02ec82c4de4f7af052397bd936eeadda, which comes from PR #1135, which was suggested by #1078.

Commit fdbd21c was introduced by PR #1086, see this little link in GitHub UI:

![where link to PR is on the commit](https://github.com/flathub/com.valvesoftware.Steam/assets/624072/a982efae-b7f8-4d3c-9fc1-432a4fcb6dd3)

The mention in question is these two entries in the issue https://github.com/ValveSoftware/steam-for-linux/issues/9383:

1. https://github.com/ValveSoftware/steam-for-linux/issues/9383#ref-pullrequest-1718615304
2. https://github.com/ValveSoftware/steam-for-linux/issues/9383#issuecomment-1556277048